### DISCUSSION TITLE
Add an audit log entry/comment when containing a host via API

### DISCUSSION THREAD

**Discussion by shwnp** [CrowdStrike/falconpy#1052] [Q&A] [answered]
Are we able to add a comment to the audit log when containing a host using the API? It's possible in the console, but I haven't seen anything in the FalconPy documentation indicating that it's possible.

**Comment by jshcodes** [comment, marked as answer]
Hi @infosec-shawn!

This functionality is not currently available via the public API. You can submit a request for the functionality to be added via the ideas portal link available from the Falcon console.

![Screenshot 2023-09-29 at 2 14 40 AM](https://github.com/CrowdStrike/falconpy/assets/74007258/bf190dc0-6945-47e5-ac6b-df7c0186e066)

Thank you for the question!

**Reply by shwnp** [reply]
Thanks for confirming!
